assets/js/editor-new.js

```diff
@@ -391,8 +391,8 @@
       imageUploadFunction: uploadImage,
       imageTexts: {
         sbInit: 'Joindre des images par glisser-déposer ou coller depuis le presse-papiers.',
-        sbOnDragEnter: 'Déposer l"image pour l"envoyer dans votre galérie',
-        sbOnDrop: 'Téléchargement d"images #images_names#',
+        sbOnDragEnter: 'Déposer l\'image pour l"envoyer dans votre galérie',
```

Comment:
L'occasion d'également corriger cette typo ?
```suggestion
        sbOnDragEnter: 'Déposer l\'image pour l"envoyer dans votre galerie',
```

Reply:
`l"envoyer` => `l\'envoyer` aussi non ?

Reply:
J'avais pas précisé que j'ai fait ça très rapidement ? 😅